### PR TITLE
Feature/10216 fss UI home screen unit test with jest

### PR DESCRIPTION
**Pull request by Rashi14748**
Added configuration files for jest framework.
Added unit tests for Header Component, Phase-Banner Component, Footer Component, Home Component.
Removed all default configurations and file/folders for Jasmine/Karma/Protractor.
Updated few cosmetic changes related to FSS-UI Home Screen

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,11 +20,7 @@ Run `ng build` to build the project. The build artifacts will be stored in the `
 
 ## Running unit tests
 
-Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
-
-## Running end-to-end tests
-
-Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
+Run `npm test` to execute the unit tests via [Jest].
```
Comment (gregzealley) [inline README.md:23]: Can we complete this link to Jest.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -44,7 +44,30 @@ stages:
...
+             testResultsFiles:  junit.xml
+            displayName: 'Publish Unit Test Results'
+          
+          - task: PublishCodeCoverageResults@1
+            inputs:
+              codeCoverageTool: 'Cobertura'
+              summaryFileLocation: '$(System.DefaultWorkingDirectory)/**/*coverage.xml'
+              reportDirectory: 'coverage/my-app/'
+              additionalCodeCoverageFiles: index.html
+            displayName: 'Code Coverage'
```
Comment (gregzealley) [inline azure-pipelines.yml:69]:
```suggestion
            displayName: 'Unit Test Code Coverage'
```

File: src/app/shared/components/fss-footer/fss-footer.component.spec.ts [hunk 3]
```diff
@@ -1,42 +1,71 @@
...
+      test('should exist', () => {
+        component = new FssFooterComponent();
+        component.ngOnInit();
+        expect(component).toBeDefined();
+      });
+              
+      test('should exist copyright statement in footer', () => {
+        component = new FssFooterComponent();
+        component.ngOnInit();
+        expect(component.text).toEqual('@ Crown copyright 2021 UK Hydrographic office');
```
Comment (gregzealley) [inline src/app/shared/components/fss-footer/fss-footer.component.spec.ts:57]: Can these values be changed to constants so they are easier to change in the future?

File: src/app/shared/components/fss-phase-banner/fss-phase-banner.component.spec.ts [hunk 4]
```diff
@@ -1,33 +1,71 @@
...
+      });
+
+      test('should exist mailto link in banner', () => {
+        component = new FssPhaseBannerComponent();
+        component.ngOnInit();
+        expect(component.link).toContain('mailto:');
+      });
+
 });
+
```
Comment (gregzealley) [inline src/app/shared/components/fss-phase-banner/fss-phase-banner.component.spec.ts:67]: Minor - few extra new lines

**Review by gregzealley — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Rashi14748) [reply README.md:23]: update link with https://jestjs.io/

File: azure-pipelines.yml (hunk 2 above)
Comment (amogh14893) [reply azure-pipelines.yml]: Done the change. 
In addition , removed **additonalCoverageFiles** from unit test step as it is not required.

File: src/app/shared/components/fss-footer/fss-footer.component.spec.ts (hunk 3 above)
Comment (Rashi14748) [reply src/app/shared/components/fss-footer/fss-footer.component.spec.ts:57]: created separate file for such static values

File: src/app/shared/components/fss-phase-banner/fss-phase-banner.component.spec.ts (hunk 4 above)
Comment (Rashi14748) [reply src/app/shared/components/fss-phase-banner/fss-phase-banner.component.spec.ts:67]: formatted documents

**Review by martyn-fewtrell — APPROVED** [review]
We need to discuss the use of fixed font sizes in style sheets in contrast to relative font sizes which are more accessible.

However do not want to hold up this pull request and will merge to main and discuss in due course.

**Pull request merged**